hadoop-ozone/integration-test/src/test/resources/ozone-site.xml

```diff
@@ -31,4 +31,9 @@
     <value>4</value>
   </property>
 
+  <property>
+    <name>ozone.scm.close.container.wait.duration</name>
+    <value>1s</value>
```

Comment:
Acceptance tests need similar low value.

https://github.com/sumitagrawl/ozone/actions/runs/4574422026/jobs/8076344130#step:5:838
https://github.com/sumitagrawl/ozone/actions/runs/4574422026/jobs/8076344350#step:5:858

Something like:

```
OZONE-SITE.XML_ozone.scm.close.container.wait.duration=1s
```

in 

```
hadoop-ozone/dist/src/main/compose/ozone/docker-config
hadoop-ozone/dist/src/main/compose/ozonesecure/docker-config
hadoop-ozone/dist/src/main/compose/ozonesecure-ha/docker-config
```

Reply:
@adoroszlai updated with 5 sec

Reply:
Thanks.